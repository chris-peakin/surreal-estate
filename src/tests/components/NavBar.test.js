import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import NavBar from "../../components/NavBarComponent";

test("navbar renders correctly", () => {
  const testRenderer = renderer.create(
    <BrowserRouter>
      <NavBar />
    </BrowserRouter>
  );
  expect(testRenderer).toMatchSnapshot();
});
