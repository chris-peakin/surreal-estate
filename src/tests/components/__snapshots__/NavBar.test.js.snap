// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar renders correctly 1`] = `
<div
  className="navbar"
>
  <img
    alt="estate-logo"
    className="logo"
    src="https://mcrcodes.s3.eu-west-2.amazonaws.com/course/surreal-estate/logo.png"
  />
  <ul
    className="navbar-links"
  >
    <li
      className="navbar-links-item"
    >
      <a
        className="navbar-links-link"
        href="/"
        onClick={[Function]}
      >
        View Properties
      </a>
    </li>
    <li
      className="navbar-links-item"
    >
      <a
        className="navbar-links-link"
        href="/add-property"
        onClick={[Function]}
      >
        Add a Property
      </a>
    </li>
    <li
      className="navbar-links-item"
    >
      <a
        className="navbar-links-link"
        href="/saved-properties"
        onClick={[Function]}
      >
        Favourite Properties
      </a>
    </li>
  </ul>
  <div
    className="login-button"
  >
    <span
      style={
        Object {
          "transition": "opacity 0.5s",
        }
      }
    >
      <button
        className="kep-login-facebook metro"
        onClick={[Function]}
        type="button"
      >
        Login
      </button>
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": Array [
              Array [
                9,
                ".kep-login-facebook{font-family:Helvetica,sans-serif;font-weight:700;-webkit-font-smoothing:antialiased;color:#fff;cursor:pointer;display:inline-block;font-size:calc(.27548vw + 12.71074px);text-decoration:none;text-transform:uppercase;transition:background-color .3s,border-color .3s;background-color:#4c69ba;border:calc(.06887vw + .67769px) solid #4c69ba;padding:calc(.34435vw + 13.38843px) calc(.34435vw + 18.38843px)}.kep-login-facebook.small{padding:calc(.34435vw + 3.38843px) calc(.34435vw + 8.38843px)}.kep-login-facebook.medium{padding:calc(.34435vw + 8.38843px) calc(.34435vw + 13.38843px)}.kep-login-facebook.metro{border-radius:0}.kep-login-facebook .fa{margin-right:calc(.34435vw + 3.38843px)}",
                "",
              ],
            ],
          }
        }
      />
    </span>
  </div>
</div>
`;
